modules/roomify/roomify_accommodation_booking/roomify_accommodation_booking.module

```diff
@@ -1612,6 +1676,107 @@ function roomify_accommodation_booking_form_bat_type_edit_form_alter(&$form, &$f
 }
 
 /**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * FORM_ID = commerce_customer_ui_customer_profile_form.
+ */
+function roomify_accommodation_booking_form_commerce_customer_ui_customer_profile_form_alter(&$form, &$form_state) {
+  if (strpos($form['#action'], 'render=references-dialog') !== FALSE) {
+    $form['profile_status']['#access'] = FALSE;
+    $form['user']['#access'] = FALSE;
+
+    if (isset($form['actions']['save_continue'])) {
+      unset($form['actions']['save_continue']);
+    }
+    if (isset($form['actions']['submit']['#suffix'])) {
+      unset($form['actions']['submit']['#suffix']);
+    }
+  }
+}
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * FORM_ID = user_register_form.
+ */
+function roomify_accommodation_booking_form_user_register_form_alter(&$form, &$form_state) {
+  $form['user_first_name']['#weight'] = -100;
+  $form['user_last_name']['#weight'] = -99;
+  $form['user_telephone']['#weight'] = -98;
+  if (strpos($form['#action'], 'render=references-dialog') !== FALSE) {
+    $form['account']['name']['#required'] = FALSE;
+    $form['account']['name']['#description'] = t('Using this (or email) the user will be able to access to the site.') . ' ' . $form['account']['name']['#description'];
```

Comment:
let's change to "Using this username (or their email) the user will be able to access the site."